src/APIFunctions/2DPrinting.js

```diff
@@ -43,9 +43,16 @@ export function parseRange(pages, maxPages) {
 
 /**
  * Print the page
- * @param {Object} data         Encoded file
- * @returns {PrintApiResponse}  Containing information for if
- *                              the page is printing
+ * @param {Object} data - Encoded file and its configurations
+ * @param {String|undefined} data.raw - Encoded file
+ * @param {Number|undefined} data.copies - Number of copies
+ * @param {String|undefined} data.sides - Sides to print:
+ * one-sided or two-sided
+ * @param {String|undefined} data.pageRanges - Pages to print
+ * @param {String|undefined} data.destination - Destination of
```

Comment:
i don't think we need this anymore as the rpc server decides, what do you think about removing it

Reply:
Me neither, we can remove it

Reply:
remove it in this pr its ok

Reply:
We need to have a separate PR in SCE-RPC to fix the function then. otherwise, it will break